### ISSUE TITLE
How might we create offline Open Educational Resources that can be used and adopted in regions without steady internet connection?

### ISSUE THREAD

**Issue by lorrainechu3n** [sparcopen/doathon#9]
[//]: # "=======================
Even if you know Github well, we suggest you read this. Anything between these lines you can leave or delete, as they won't display anyway when you post (you can check this via Preview changes). They're here to help you complete issues quickly and in a way that will help other participants. If you're posting a new project, or challenge. We suggest you fill out the Google Forms first.
============================"

## At a glance

[//]: # "=======================
Please paste the metadata you received after submitting your project or challenge in your Google Form exactly as we sent it to you. You can delete what's there now, it's just there
============================"

Submission Name: Open Educational Resources that can be used and adopted in regions without steady internet connection?

Contact Lead: @lorrainechu3n (This is an example submission) 

Region: #Global

Issue Area: #OpenEducation 

Issue Type: #Challenge

## Description
Although one of the goals of the Open Education movement is to reduce barriers to accessing education, many OER remain reliant on digital infrastructure, and these educational materials remain inaccessible to those without wifi at home, or in regions where internet is more expensive and primarily accessed on mobile. For this challenge statement, we are asking how can we create OER that can be adapted offline, or that take into account the digital divide.

##  How can others contribute?
// More details coming soon — for now, leave any relevant comments, readings, or links on this thread! 

### end. 
[//]: # "=======================
After posting your issue, the real work begins. Next you might want to:
   Tweet a link to this issue with #opencon so others can join in
   Make another issue to involve people in your work - remember to use your metadata
   Come back from time to time and update the community on your project.
You'll get an email update whenever someone interacts with your issue.
============================"

_This post is part of the OpenCon 2017 Do-A-Thon. Not sure what's going on, [head here](doathon.opencon2017.org)_

**Comment by illyz** [comment]
@lorrainechu3n  I'd be interested in working on this! Has anything started yet?

**Comment by nisstyre56** [comment]
I would suggest looking at building a Progressive Web App (PWA). They enable users to use the app on their phone even with an unreliable (or absent) connection. There's a good overview here, [https://developers.google.com/web/progressive-web-apps/](https://developers.google.com/web/progressive-web-apps/)

Also, many of the people using older or less expensive phones use [Opera Mini](https://www.opera.com/mobile) to browser the web, and it will do things that cause a lot of modern web apps to stop working, such as removing JavaScript and aggressively compressing images and pages. So I think a good approach is to build sites that do not require the use of JavaScript, or very minimal use. [Filament Group](https://www.filamentgroup.com/lab/) has tons of articles on best practices for building static websites that load very efficiently. So I would highly recommend skimming through some of the articles, particularly about accessibility and optimizing bandwidth.

**Comment by nisstyre56** [comment]
This is a good article that explains why Opera Mini is so prevalent for people on less reliable connections, [https://blogs.opera.com/news/2015/11/opera-mini-saves-african-mobile-internet-users-half-a-billion-us-2/](https://blogs.opera.com/news/2015/11/opera-mini-saves-african-mobile-internet-users-half-a-billion-us-2/)

**Comment by nisstyre56** [comment]
I would love to be involved in this somehow, but I was wondering whether the intention of this was to just collect information on how to build more accessible OER, or to work on some kind of guide for how to do this (for publishers, etc).

**Comment by chartgerink** [comment]
Would it be possible that the start of decentralized internet protocols such as [Dat](https://datproject.org) provide the perfect way of doing this?

They are currently a bit more difficult to use for non-techies, maybe, but with Beaker browser this is becoming more reasonable by the day. It works a bit like torrents but with more stable addresses. If someone near you has the content, you can get it from them (via LAN route, but also USB for example) and serve it yourself (no server necessary).

Tagging @daniellecrobinson who might be interested in this for her work for [Code for Science & Society](https://codeforscience.org/).

**Comment by nisstyre56** [comment]
@chartgerink WebRTC might provide a good way of implementing that too. You still need a signaling server (in order to coordinate everything) but it does not require the connection to be low latency and uses a small amount of bandwidth.

**Comment by nisstyre56** [comment]
@chartgerink @lorrainechu3n @illyz made a discourse server for us to brainstorm this some more https://discord.gg/FsqNCkd

**Comment by Ongironald** [comment]
Greetings I am Ronald Emojong, I am very interesting in helping share insights and where possible participate. What is pending on this project, what has currently been taking place in 2018 so far. Regards Ronald

**Comment by jamshidhashimi** [comment]
There are initiatives like [RACHEL](http://rachelfriends.org/), [Rumie](https://www.rumie.org/) and others, but it will be really good to explore those solutions, identify their strengths and weaknesses and propose solutions so that can be adapted and implemented. 

@JosephMcArthur do we have this as a Do-a-thon/Unconference this year? I am really interested to be part of this discussion and work with others in coming up with recommendations and solutions. Thank you.

**Comment by samuk** [comment]
Hi I'm just starting a project to sell (not for profit) 32Gb SD cards into African markets.

The SD cards will be full of OER & Open Apps.

Plenty to do if anyone is still interested in this approach: https://github.com/sparcopen/doathon/issues/85
